lib/schema.js

```diff
@@ -3,7 +3,7 @@
 const Joi = require('joi');
 
 exports.options = Joi.object({
-    models: Joi.array().items(Joi.func().class()),
+    models: Joi.array().items(Joi.func().class()).min(1),
```

Comment:
Is there any reason not to no-op when `models` in an empty array?

Reply:
Possible -- I sort-of had it this way by responding with a success message that there were no models provided but I figured this would help the user figure out what's going wrong quicker if an error is thrown -- what do you think a no-op should look like? Return anything?